### ISSUE TITLE
Ability to copy paste requests between folders

### ISSUE THREAD

**Issue by CosminGramada** [riok/Kreya#148]
**Is your feature request related to a problem? Please describe.**
It would be great to be able to copy-paste existing requests between folders.
One scenario would be copying and pasting prerequisite requests that are common between tests.

Currently, as a workaround, I'm copy-pasting on the disk level but it would be easier to just do it from the application itself.

**Describe the solution you'd like**
Copy-paste requests (or even better entire folders) between folders.

**Comment by CosminGramada** [comment]
I saw the duplicate feature, but when you need to use it often it becomes time-consuming because you need to drag and drop it and then also rename due to the `-copy` appended text

**Comment by CommonGuy** [comment]
Yes, I see your point. We should probably implement shortcuts such as CTRL+C for both operations and folders to make this easier.
